### DISCUSSION TITLE
SONOFF socket not reachable

### DISCUSSION THREAD

**Discussion by HeliosAnavatis** [arendst/Tasmota#16886] [Q&A]
Had this problem on 10.x firmware, so upgraded to 12.2.0 but it hasn't made a difference.

I have three SONF S26R2ZBTPG sockets connected to a Tasmotised SONOFF ZBBridge.  

The first two are 'Charger' and 'Heater', which both work fine and I can turn on and off via OpenHAB.  The third, though will not make itself fully available.  I can 'pair' it and see it on the map:

`15:25:36.467 ZIG: sending `ZbMap 0x0000`
15:25:36.615 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0x0000","Status":0,"StatusMessage":"SUCCESS","Total":3,"Start":1,"Map":[{"Device":"0x58FE","Name":"Printer","DeviceType":"Router","RxOnWhenIdle":true,"Relationship":"Sibling","PermitJoin":null,"Depth":15,"LinkQuality":232},{"Device":"0x80CC","Name":"Charger","DeviceType":"Router","RxOnWhenIdle":true,"Relationship":"Sibling","PermitJoin":null,"Depth":15,"LinkQuality":161},{"Device":"0xF6F8","Name":"Heater","DeviceType":"Router","RxOnWhenIdle":true,"Relationship":"Sibling","PermitJoin":null,"Depth":15,"LinkQuality":168}]}}
15:25:38.489 ZIG: sending `ZbMap 0x80CC`
15:25:38.578 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0x80CC","Name":"Charger","Status":0,"StatusMessage":"SUCCESS","Total":4,"Start":1,"Map":[{"Device":"0xF6F8","Name":"Heater","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":45},{"Device":"0x0000","DeviceType":"Coordinator","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":122}]}}
15:25:38.619 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0x80CC","Name":"Charger","Status":0,"StatusMessage":"SUCCESS","Total":4,"Start":3,"Map":[{"Device":"0xF9A5","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":0},{"Device":"0x58FE","Name":"Printer","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":7}]}}
15:25:40.479 ZIG: sending `ZbMap 0xF6F8`
15:25:40.565 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0xF6F8","Name":"Heater","Status":0,"StatusMessage":"SUCCESS","Total":3,"Start":1,"Map":[{"Device":"0x0000","DeviceType":"Coordinator","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":128},{"Device":"0x58FE","Name":"Printer","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":1}]}}
15:25:40.601 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0xF6F8","Name":"Heater","Status":0,"StatusMessage":"SUCCESS","Total":3,"Start":3,"Map":[{"Device":"0x80CC","Name":"Charger","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":48}]}}
15:25:42.476 ZIG: sending `ZbMap 0x58FE`
15:25:42.563 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0x58FE","Name":"Printer","Status":0,"StatusMessage":"SUCCESS","Total":3,"Start":1,"Map":[{"Device":"0x0000","DeviceType":"Coordinator","RxOnWhenIdle":null,"Relationship":"Parent","PermitJoin":null,"Depth":0,"LinkQuality":172},{"Device":"0xF6F8","Name":"Heater","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":1}]}}
15:25:42.598 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbMap":{"Device":"0x58FE","Name":"Printer","Status":0,"StatusMessage":"SUCCESS","Total":3,"Start":3,"Map":[{"Device":"0x80CC","Name":"Charger","DeviceType":"Router","RxOnWhenIdle":null,"Relationship":"Sibling","PermitJoin":null,"Depth":255,"LinkQuality":5}]}}
15:25:48.488 ZIG: ZbMap done
`

But when I try to turn the power to the Printer socket on, I get:

`15:13:29.209 CMD: ZbSend {"Device":"Printer","Send":{"Power":"On"}}
15:13:29.216 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
15:13:31.452 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbReceived":{"0x58FE":{"Device":"0x58FE","Name":"Printer","Reachable":false}}}`

The other two work fine:

`15:28:13.010 CMD: ZbSend {"Device":"Heater","Send":{"Power":"On"}}
15:28:13.017 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
15:28:13.730 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbReceived":{"0xF6F8":{"Device":"0xF6F8","Name":"Heater","Power":1,"Endpoint":1,"LinkQuality":68}}}`

ZbInfo shows me:

`15:29:32.632 CMD: ZbInfo3 1
15:29:32.641 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbInfo":{"0x80CC":{"Device":"0x80CC","Name":"Charger","IEEEAddr":"0x00124B0024C11827","ModelId":"S26R2ZB","Manufacturer":"SONOFF","Endpoints":[1,13,242],"Config":["O01"],"Power":0,"Reachable":true,"LastSeen":234,"LastSeenEpoch":1666535138,"LinkQuality":71}}}
15:29:32.643 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbInfo":"Done"}
15:29:35.576 CMD: ZbInfo3 2
15:29:35.585 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbInfo":{"0xF6F8":{"Device":"0xF6F8","Name":"Heater","IEEEAddr":"0x00124B0024C1179E","ModelId":"S26R2ZB","Manufacturer":"SONOFF","Endpoints":[1,13],"Config":["O01"],"Power":0,"Reachable":true,"LastSeen":75,"LastSeenEpoch":1666535300,"LinkQuality":76}}}
15:29:35.588 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbInfo":"Done"}
15:29:37.902 CMD: ZbInfo3 3
15:29:37.909 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbInfo":{"0x58FE":{"Device":"0x58FE","Name":"Printer","IEEEAddr":"0x00124B0024C10F89","Endpoints":[13,1,242],"Config":[],"Reachable":false,"LastSeen":235,"LastSeenEpoch":1666535142,"LinkQuality":118}}}
15:29:37.914 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbInfo":"Done"}`

There's clearly something different about the third socket (and I've tried a fourth with exactly the same results!)

Can anyone please point me in the right direction to solve this issue?  Thanks in advance.

**Comment by s-hadinger** [comment]
Please set `Weblog 3` when sending `ZbSend {"Device":"Printer","Send":{"Power":"On"}}`

From the short log, it seems that the device is not reachable

Could you also send the logs for the pairing of `Printer` (with `WebLog 2`)

**Comment by HeliosAnavatis** [comment]
@s-hadinger  - Thanks for responding so quickly.  Here's the log:

Edit: changing to triple ticks, otherwise it's hard to read

```
18:28:44.826 CMD: Weblog 3
18:28:44.830 MQT: stat/tasmota_9DBEA7/RESULT = {"WebLog":3}
18:28:45.550 CFG: Saved to flash at FD, Count 104, Bytes 4096
18:29:02.386 CMD: ZbSend {"Device":"Printer","Send":{"Power":"On"}}
18:29:02.387 SRC: WebConsole from 192.168.1.102
18:29:02.389 CMD: Grp 0, Cmd 'ZBSEND', Idx 1, Len 42, Pld -99, Data '{"Device":"Printer","Send":{"Power":"On"}}'
18:29:02.392 ZIG: guessing endpoint 13
18:29:02.393 ZigbeeZCLSend device: 0x58FE, endpoint:13, cluster:0x0006, cmd:0x01, send:""
18:29:02.394 ZIG: ZbEZSPSend 340000FE5804010600010D40010000280103112801
18:29:02.399 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
18:29:02.477 ZIG: {"ZbEZSPReceived":"34000076"}
18:29:02.495 ZIG: {"ZbEZSPReceived":"3F0000FE5804010600010D4001000076010000"}
18:29:02.598 ZigbeeZCLSend device: 0x58FE, endpoint:13, cluster:0x0006, cmd:0x00, send:"0000"
18:29:02.599 ZIG: ZbEZSPSend 340000FE5804010600010D400100002901050029000000
18:29:02.656 ZIG: {"ZbEZSPReceived":"34000077"}
18:29:02.675 ZIG: {"ZbEZSPReceived":"3F0000FE5804010600010D4001000077010000"}
18:29:04.634 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbReceived":{"0x58FE":{"Device":"0x58FE","Name":"Printer","Reachable":false}}}
```

And to compare with a working socket:

```
18:30:25.041 CMD: ZbSend {"Device":"Heater","Send":{"Power":"On"}}
18:30:25.042 SRC: WebConsole from 192.168.1.102
18:30:25.044 CMD: Grp 0, Cmd 'ZBSEND', Idx 1, Len 41, Pld -99, Data '{"Device":"Heater","Send":{"Power":"On"}}'
18:30:25.046 ZIG: guessing endpoint 1
18:30:25.048 ZigbeeZCLSend device: 0xF6F8, endpoint:1, cluster:0x0006, cmd:0x01, send:""
18:30:25.049 ZIG: ZbEZSPSend 340000F8F604010600010140010000100103111001
18:30:25.053 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
18:30:25.126 ZIG: {"ZbEZSPReceived":"34000078"}
18:30:25.140 ZIG: {"ZbEZSPReceived":"3F0000F8F60401060001014001000078010000"}
18:30:25.147 ZIG: {"ZbEZSPReceived":"450000040106000101000000002CA8C6F8F6FFFF0708190A00001001"}
18:30:25.149 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0006","srcaddr":"0xF6F8","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":76,"securityuse":0,"seqnumber":44,"fc":"0x08","frametype":0,"direction":1,"disableresp":0,"manuf":"0x0000","transact":25,"cmdid":"0x0A","payload":"00001001"}}
18:30:25.151 ZigbeeZCLSend device: 0xF6F8, endpoint:1, cluster:0x0006, cmd:0x0B, send:"0A00"
18:30:25.152 ZIG: ZbEZSPSend 340000F8F60401060001010000000019010510190B0A00
18:30:25.153 ZIG: Battery auto-probe `ZbSend {"Device":"0xF6F8","Read":{"BatteryPercentage":true,"BatteryVoltage":true}}`
18:30:25.154 ZIG: ZbZCLRawReceived: {"0xF6F8":{"0006/0000":1,"Endpoint":1,"LinkQuality":76}}
18:30:25.158 ZIG: {"ZbEZSPReceived":"450000040106000101000000002D9CC3F8F6FFFF07081A0A00002001"}
18:30:25.160 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0006","srcaddr":"0xF6F8","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":68,"securityuse":0,"seqnumber":45,"fc":"0x08","frametype":0,"direction":1,"disableresp":0,"manuf":"0x0000","transact":26,"cmdid":"0x0A","payload":"00002001"}}
18:30:25.161 ZigbeeZCLSend device: 0xF6F8, endpoint:1, cluster:0x0006, cmd:0x0B, send:"0A00"
18:30:25.162 ZIG: ZbEZSPSend 340000F8F6040106000101000000001A0105101A0B0A00
18:30:25.164 ZIG: ZbZCLRawReceived: {"0xF6F8":{"0006/0000":1,"Endpoint":1,"LinkQuality":68}}
18:30:25.167 ZIG: {"ZbEZSPReceived":"34000079"}
18:30:25.169 ZigbeeZCLSend device: 0xF6F8, endpoint:1, cluster:0x0001, cmd:0x00, send:"20002100"
18:30:25.169 ZIG: ZbEZSPSend 340000F8F60401010001014001000011010700110020002100
18:30:25.171 ZIG: NAK, resending packet 0
18:30:25.173 ZIG: {"ZbEZSPReceived":"3F0000F8F60401060001010000000079010000"}
18:30:25.179 ZIG: {"ZbEZSPReceived":"3400007A"}
18:30:25.181 ZIG: NAK, resending packet 1
18:30:25.191 ZIG: {"ZbEZSPReceived":"3400007B"}
18:30:25.195 ZIG: {"ZbEZSPReceived":"3F0000F8F6040106000101000000007A010000"}
18:30:25.213 ZIG: {"ZbEZSPReceived":"3F0000F8F6040101000101400100007B010000"}
18:30:25.221 ZIG: {"ZbEZSPReceived":"450000040101000101000100002E9CC3F8F6FFFF09181101200086210086"}
18:30:25.222 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0001","srcaddr":"0xF6F8","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":68,"securityuse":0,"seqnumber":46,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":17,"cmdid":"0x01","payload":"200086210086"}}
18:30:25.224 ZIG: ZbZCLRawReceived: {"0xF6F8":{"Endpoint":1,"LinkQuality":68}}
18:30:25.271 ZigbeeZCLSend device: 0xF6F8, endpoint:1, cluster:0x0006, cmd:0x00, send:"0000"
18:30:25.272 ZIG: ZbEZSPSend 340000F8F6040106000101400100001201050012000000
18:30:25.330 ZIG: {"ZbEZSPReceived":"3400007C"}
18:30:25.347 ZIG: {"ZbEZSPReceived":"3F0000F8F6040106000101400100007C010000"}
18:30:25.354 ZIG: {"ZbEZSPReceived":"450000040106000101000000002F9CC3F8F6FFFF081812010000001001"}
18:30:25.356 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0006","srcaddr":"0xF6F8","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":68,"securityuse":0,"seqnumber":47,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":18,"cmdid":"0x01","payload":"0000001001"}}
18:30:25.358 ZIG: ZbZCLRawReceived: {"0xF6F8":{"0006/0000":1,"Endpoint":1,"LinkQuality":68}}
18:30:25.763 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbReceived":{"0xF6F8":{"Device":"0xF6F8","Name":"Heater","Power":1,"Endpoint":1,"LinkQuality":68}}}
```

Attempting to re-pair with `Weblog 2`:
```
18:57:20.635 CMD: zbleave Printer
18:57:20.641 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbLeave":"Done"}
18:57:20.769 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B12000203FFFF"}
18:57:27.119 CMD: zbforget Printer
18:57:27.124 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbForget":"Done"}
18:57:29.236 ZIG: Zigbee Devices Data saved in File System (94 bytes)
18:57:56.575 CMD: ZbPermitJoin 1
18:57:56.580 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":21,"Message":"Pairing mode enabled"}}
18:57:56.582 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbPermitJoin":"Done"}
18:58:04.659 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B120001000000"}
18:58:04.664 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
18:58:04.721 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","PowerSource":true,"ReceiveWhenIdle":true,"Security":false}}
18:58:04.767 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x0D","0x01","0xF2"]}}
18:58:04.792 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:04.804 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:06.784 ZIG: Zigbee Devices Data saved in File System (122 bytes)
18:58:13.751 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B120001000000"}
18:58:13.755 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
18:58:13.807 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","PowerSource":true,"ReceiveWhenIdle":true,"Security":false}}
18:58:13.853 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x0D","0x01","0xF2"]}}
18:58:13.879 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:13.911 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:28.176 MQT: tele/tasmota_9DBEA7/STATE = {"Time":"2022-10-23T18:58:28","Uptime":"0T04:30:13","UptimeSec":16213,"Vcc":3.510,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":3,"Wifi":{"AP":1,"SSId":"********","BSSId":"************","Channel":11,"Mode":"11n","RSSI":100,"Signal":-38,"LinkCount":3,"Downtime":"0T00:00:10"}}
18:58:32.818 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B120001000000"}
18:58:32.822 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
18:58:32.883 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","PowerSource":true,"ReceiveWhenIdle":true,"Security":false}}
18:58:32.927 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x0D","0x01","0xF2"]}}
18:58:32.954 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:32.982 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:40.494 CMD: ZbSend {"Device":"0x58FE","Send":{"Power":"On"}}
18:58:40.501 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
18:58:41.934 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B120001000000"}
18:58:41.941 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
18:58:41.993 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","PowerSource":true,"ReceiveWhenIdle":true,"Security":false}}
18:58:42.038 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x0D","0x01","0xF2"]}}
18:58:42.064 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:42.089 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:51.003 ZIG: {"ZbEZSPReceived":"2400FE58890FC124004B120001000000"}
18:58:51.007 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
18:58:51.059 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x58FE","PowerSource":true,"ReceiveWhenIdle":true,"Security":false}}
18:58:51.104 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x0D","0x01","0xF2"]}}
18:58:51.131 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:51.156 ZIG: {"ZbEZSPReceived":"9B00890FC124004B120006"}
18:58:56.636 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":20,"Message":"Pairing mode disabled"}}
18:59:19.170 CMD: ZbSend {"Device":"0x58FE","Send":{"Power":"On"}}
18:59:19.177 MQT: stat/tasmota_9DBEA7/RESULT = {"ZbSend":"Done"}
18:59:20.525 ZIG: {"ZbEZSPReceived":"24007236890FC124004B12000100F8F6"}
18:59:20.528 MQT: tele/tasmota_9DBEA7/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024C10F89","ShortAddr":"0x3672","ParentNetwork":"0xF6F8","JoinStatus":1,"Decision":0}}
18:59:21.427 MQT: tele/tasmota_9DBEA7/SENSOR = {"ZbReceived":{"0x58FE":{"Device":"0x58FE","Reachable":false}}}
18:59:23.540 ZIG: Zigbee Devices Data saved in File System (138 bytes)
```

**Reply by s-hadinger** [reply]
The process of joining fails, il looks like the device is not happy and tries to register again and again. I have no clue why this is happening. I'm afraid only a Zigbee sniffer can tell us why this happens. Or the firmware of those devices is faulty.

**Reply by HeliosAnavatis** [reply]
Oh dear :( - Well at least I know.  Weird that two other identical plugs work, and these work on another Zb network.  I have some more plugs I can try, so I'll have a go swapping things round and if that doesn't work, I'll see if I can reflash them as a last resort - I vaguely remember seeing that there's open firmware available for them somewhere.

Thanks for replying, though - much appreciated.

**Reply by enforcebeautifulcode** [reply]
@HeliosAnavatis try endpoint 1.

The working socket is guessing endpoint 1 and the non working socket is attempting to use endpoint 13.
